src/main/java/io/jenkins/plugins/coverage/CoverageProcessor.java

```diff
@@ -97,6 +105,7 @@ public void performCoverageReport(List<CoverageReportAdapter> reportAdapters, Li
         HealthReport healthReport = processThresholds(results, globalThresholds);
 
 
+        setDiffInCoverageForChangeRequest(coverageReport);
```

Comment:
Should we make it a configurable option?

Reply:
Yes, I think it would be better to make it configurable. Will change it.

Reply:
@tandibar I think we should config it in Publisher level :)

Reply:
@cizezsy Please, check